### PR TITLE
Aim Trainer

### PR DESCRIPTION
**Pull request by avimoti1234**
If there is anything you want, let me know

### CODE REVIEW

**Comment by sree-hari-s** [comment]
very nicely done can you please include a readme file just containing a brief description of your work 

and I have created a dedicated directory for tkinter projects so you can change you file location also.

**Comment by avimoti1234** [comment]
Hey, 
I have done all that:)
Would be glad if you merge it.

**Comment by avimoti1234** [comment]
I've deleted some .xml files that I accidentally pushed

**Comment by avimoti1234** [comment]
And add the hacktoberfest label:)

File: Projects/Tkinter-Projects/Aim Trainer/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+library.
+
+
+## How to Run
+
+1. Clone the repository:
+
+   ```bash
+   git clone <repository_url>
+   cd Aim Trainer
```
Comment (sree-hari-s) [inline Projects/Tkinter-Projects/Aim Trainer/README.md:20]: everyone seems to be copy-pasting previous work, isn't it?

File: Projects/Tkinter-Projects/Aim Trainer/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+tkinter
```
Comment (sree-hari-s) [inline Projects/Tkinter-Projects/Aim Trainer/requirements.txt:1]: both are in-buit packages

**Review by sree-hari-s — APPROVED** [review]

**Review by sree-hari-s — CHANGES_REQUESTED** [review]
You seem to have changed a lot of file please remove those changes.

**Comment by avimoti1234** [comment]
I have only changed the file's day number because before you told me to move the project to the Tkinter folder it was located in the intermediate folder with the day number so I had to decrement all of the files day number

**Comment by sree-hari-s** [comment]
> I have only changed the file's day number because before you told me to move the project to the Tkinter folder it was located in the intermediate folder with the day number so I had to decrement all of the files day number

You didn't pull the latest changes that's why it happened

**Comment by avimoti1234** [comment]
I have just done that again and I don't recognize any difference.
Can you explain me because everything looks fine to me

**Comment by sree-hari-s** [comment]
I had changed few folders to here and there so your PR has 51 changed files so a conflict is happening

**Review by avimoti1234 — COMMENTED** [review]
Is that ok?

**Comment by avimoti1234** [comment]
How can I solve it?

**Comment by sree-hari-s** [comment]
It would be better if you pull the latest  code and raise a PR again.

**Pull request closed without merging**